moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -114,8 +121,10 @@ const ProjectsListViewTable = () => {
 
   const { query: projectListViewQuery, exportQuery } = useGetProjectListView({
     columnsToReturn: columnsToReturnInQuery,
-    exportColumnsToReturn: Object.keys(PROJECT_LIST_VIEW_EXPORT_CONFIG),
-    exportConfig: PROJECT_LIST_VIEW_EXPORT_CONFIG,
+    exportColumnsToReturn:
+      columnDownloadOption === "visible"
+        ? columnsToReturnInQuery
+        : Object.keys(PROJECT_LIST_VIEW_EXPORT_CONFIG),
```

Comment:
If the user selected the "visible" option then only export the columns that are being returned in the table query, otherwise export all the columns in the export config